src/foam/u2/view/ReadOnlyEnumView.js

```diff
@@ -56,14 +56,14 @@ foam.CLASS({
         .style({
           'background-color': background,
           'color': color,
-          'border-color': background == '#FFFFFF' || ! background ? color : background
+          'border-color': background != '#FFFFFF' || background ? background : color
```

Comment:
My previous comment was actually wrong, because I didn't read the expression before. So this isn't actually any better (or worse). Sorry.